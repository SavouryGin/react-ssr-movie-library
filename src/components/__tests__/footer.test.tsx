import Footer from 'components/footer/index';
import React from 'react';
import { render, screen } from '@testing-library/react';

describe('Footer component:', () => {
  it('Should matche the snapshot', () => {
    const { asFragment } = render(<Footer className='test-classname' />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('Should render the footer container', () => {
    render(<Footer className='test-classname' />);
    expect(screen.getByRole('contentinfo')).toBeInTheDocument();
  });
});
