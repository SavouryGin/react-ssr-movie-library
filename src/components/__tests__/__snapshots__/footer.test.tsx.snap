// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component: Should matche the snapshot 1`] = `
<DocumentFragment>
  <footer
    class="test-classname"
  >
    <span>
      <strong>
        netflix
      </strong>
      roulette
    </span>
  </footer>
</DocumentFragment>
`;
